node/Dockerfile-windows

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2023 Tigera, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG WINDOWS_VERSION
+
+FROM mcr.microsoft.com/windows/nanoserver:${WINDOWS_VERSION}
```

Comment:
should be able to use `mcr.microsoft.com/oss/kubernetes/windows-host-process-containers-base-image:v1.0.0` since it is HPC image.  This will make it OS agnostic and faster download: https://github.com/microsoft/windows-host-process-containers-base-image

Reply:
@jsturtevant I didn't know about that image, this is awesome! Thanks, will use it!